### PR TITLE
Enhancement/style overhaul

### PR DESCRIPTION
**Pull request by acheung-94**

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -3,6 +3,9 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/x-icon" href="https://pet-portal-assets.s3.us-west-1.amazonaws.com/paw-svgrepo-com.svg" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=DM+Serif+Display:ital@0;1&family=Montserrat:ital,wght@0,100..900;1,100..900&family=Raleway:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
```
Comment (ChristopherJTrent) [inline frontend/index.html]: the link tags aren't necessary when using an import at-rule in your CSS
```suggestion
```

**Review by ChristopherJTrent — CHANGES_REQUESTED** [review]
One requested change, otherwise LGTM

**Review by ChristopherJTrent — APPROVED** [review]
LGTM, just waiting on review from Sam

**Review by sam-kim99 — APPROVED** [review]

**Comment by ChristopherJTrent** [comment]
:tada:

**Pull request merged**